src/Api/Controller/ListUsersController.php

```diff
@@ -78,6 +79,8 @@ protected function data(ServerRequestInterface $request, Document $document)
         $offset = $this->extractOffset($request);
         $load = $this->extractInclude($request);
 
+        $document->addMeta('totalCount', User::count());
```

Comment:
I'll open this as a separate comment. On a more serious note, I'm a bit wary about making this modification. However, we're currently improperly using the 5th argument of `addPaginationLinks`: we should be providing a total, which would put together a "last" item in the JSON:API links. That change should be done in a separate issue/PR though.

For now, maybe the API Payload would be a better place to pass this?

Reply:
Even with the `last` item, surely that won't provide the total number of users directly, right?

We could extract it from the URL, but imo that's extra processing that should be done server-side instead of client-side, unless I'm misunderstanding what you meant.

Reply:
Yeah, fair. My concern is inconsistency with other endpoints, and that the REST response is public API. That's why I prefer to have it in AdminPayload (https://github.com/flarum/core/blob/ea291508ab954a04682927ebe3bfafbfb584e752/src/Admin/Content/AdminPayload.php#L22-L22), as that's not considered public.

Reply:
Done. I opted to use the same API that `flarum/statistics` uses (`statistics.users.total`), because it's a core ext and will override the data set in `AdminPayload.php`, and provides one single source of truth, rather than duplicating data.